src/app/modules/card-cluster/card-cluster.component.css

```diff
@@ -23,10 +25,24 @@
     overflow: hidden;
 }
 
+.cardHeight.active  {
+    height: 72px;
+}
+
 .ds-text {
     margin-top: 0;
 }
 
+.cardClusterContent {
+    padding-top: 4px;
+    pointer-events: none;
+    transition: all 200ms cubic-bezier(.25,.8,.25,1);
```

Comment:
Does this need legacy browser support (mop-transition, webkit-transition, etc)?

Reply:
It seems like all modern browsers as of 2012 support the normal transition css property (https://caniuse.com/#feat=css-transitions)

It seems like the current standard is to put the transition properties in side of a @supports decorator (https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-webkit-transition) so I added that in